file-formats/sito/sito-v1.json

```
@@ -0,0 +1,893 @@
+                ],
+                "enumDescriptions": [
+                  "Yes",
+                  "No"
+                ]
+              },
+              "sitnobjstructype": {
+                "title": "Situation Object Structure Type",
+                "description": "Object Structure Type",
+                "type": "string",
+                "enum": [
+                  "cds",
+                  "inMemory"
+                ],
+                "enumDescriptions": [
+                  "CDS",
+                  "In Memory"
+                ]
+              },
+              "sitnobjstrucscope": {
+                "title": "Source",
+                "description": "Object Source: SAP or Customer",
+                "type": "string",
+                "enum": [
+                  "sap",
+                  "customer"
+                ],
+                "enumDescriptions": [
+                  "SAP",
+                  "Customer"
+                ]
+              },
+              "sitnobjstrucname": {
+                "description": "DD: Name of a structured object (e.g. entity, context)",
+                "type": "string",
+                "maxLength": 30
+              },
+              "sitnobjstrucid": {
+                "title": "Situation Object Structure ID",
+                "description": "Situation Object Structure ID",
+                "type": "string",
+                "maxLength": 32
+              },
+              "sit2ObjStrucT": {
+                "title": "List of SIT2_OBJ_STRUC_T",
+                "description": "List of SIT2_OBJ_STRUC_T",
+                "type": "array",
+                "items": {
+                  "title": "SIT2_OBJ_STRUC_T",
+                  "description": "SIT2_OBJ_STRUC_T",
+                  "type": "object",
+                  "properties": {
+                    "sitnobjstrucdesc": {
+                      "title": "Situation Object Structure Description",
+                      "description": "Situation Object Structure Description",
+                      "type": "string",
+                      "maxLength": 255
+                    },
+                    "language": {
+                      "title": "Language Key",
+                      "description": "Language Key",
+                      "type": "string",
+                      "minLength": 2,
+                      "maxLength": 2,
+                      "pattern": "^[a-z]+$"
+                    }
+                  },
+                  "additionalProperties": false
+                }
+              },
+              "sit2ObjStrSk": {
+                "title": "List of SIT2_OBJ_STR_SK",
+                "description": "List of SIT2_OBJ_STR_SK",
+                "type": "array",
+                "items": {
+                  "title": "SIT2_OBJ_STR_SK",
+                  "description": "SIT2_OBJ_STR_SK",
+                  "type": "object",
+                  "properties": {
+                    "sitnobjectfieldorder": {
```

Just one example, how I would rename the fields. In the ABAP interface it would be renamed to `field_order` instead of `sitnobjectfieldorder`:

```suggestion
                    "fieldOrder": {
```